packages/optimistic-oracle/index.js

```diff
@@ -148,6 +153,11 @@ async function Poll(callback) {
       // Type of "Oracle" set for this network's Finder, default is "Voting". Other possible types include "SinkOracle",
       //  "OracleChildTunnel", and "MockOracleAncillary"
       oracleType: process.env.ORACLE_TYPE ? process.env.ORACLE_TYPE : "Voting",
+      // Type of "OptimisticOracle" to load in client, default is "OptimisticOracle". The only other possible type is
+      // "SkinnyOptimisticOracle."
+      optimisticOracleType: process.env.OPTIMISTIC_ORACLE_TYPE
```

Comment:
could do the validation here

Reply:
We should do validation for the `oracleType` as well, will review

Reply:
Actually we should make that a different PR so it doesn't break production bots